sjsonnet/src/sjsonnet/Std.scala

```diff
@@ -702,92 +733,6 @@ class Std {
     }
   }
 
-  private object SetInter extends Val.Builtin3("setInter", "a", "b", "keyF", Array(null, null, Val.False(dummyPos))) {
```

Comment:
Is there another implementation of SetInter that is replacing this?

Reply:
oh yeah, there's one. Line 1188